uw-frame-components/portal/widgets/controllers.js

```diff
@@ -181,6 +181,13 @@ define(['angular'], function(angular) {
       if (!$scope.config.showShowing) {
         $scope.config.showShowing = false;
       }
+      // Set sensible fallbacks in case widget config would create display problems.
+      if ($scope.config.lim > 6) {
+        $scope.config.lim = 6;
+      }
+      if ($scope.config.titleLim > 50) {
```

Comment:
Fair enough,
It would be good to have a discussion about long term configuration management.
Mixing configuration validation into business logic into is a easy stop gap to add, but can be difficult to track down an remove down the road.